README.md

```diff
@@ -5,8 +5,10 @@
 * [jdk8](https://github.com/keeganwitt/docker-gradle/blob/master/jdk8/Dockerfile)
 * [jdk11](https://github.com/keeganwitt/docker-gradle/blob/master/jdk11/Dockerfile)
 * [jdk11-alpine](https://github.com/keeganwitt/docker-gradle/blob/master/jdk11-alpine/Dockerfile)
-* [jdk17, latest](https://github.com/keeganwitt/docker-gradle/blob/master/jdk17/Dockerfile)
-* [jdk17-alpine, alpine](https://github.com/keeganwitt/docker-gradle/blob/master/jdk17-alpine/Dockerfile)
+* [jdk17](https://github.com/keeganwitt/docker-gradle/blob/master/jdk17/Dockerfile)
+* [jdk17-alpine](https://github.com/keeganwitt/docker-gradle/blob/master/jdk17-alpine/Dockerfile)
+* [jdk18, latest](https://github.com/keeganwitt/docker-gradle/blob/master/jdk18/Dockerfile)
```

Comment:
Since 18 is not an LTS release, we wouldn't point the latest tag to that.

Reply:
Fixed!